frontend/src/index.html

```diff
@@ -299,7 +175,7 @@ <h3 class="flex-grow break-words font-semibold" title="{{title}}">{{#helpers.hig
         </div>
         <div id="pagination" class="my-2 w-full"></div>
         <p id="under-progress" class="hidden p-2 text-center text-gray-500 text-sm">
-          This Archive is under progress and will be completed by 2023. Progressively, new recordings will be added regularly. If you don’t find the audio you are looking for, we request you to check
+          This Archive is under progress and will be completed by 2023. Progressively, new recordings will be added regularly. If you don�t find the audio you are looking for, we request you to check
```

Comment:
Some strange symbol appeared instead of the quotation mark. Please revert this change.